### ISSUE TITLE
Train Plant Species Recognition Model (MobileNet V2)

### ISSUE THREAD

**Issue by hxw0320** [spsaswat/plantdis#83]
## Description:
 - Use the prepared plant species recognition dataset, where each image is labeled only with its plant species (disease labels are ignored).
 - Implement the MobileNet V2 architecture as the backbone for the plant species recognition model to ensure lightweight and efficient inference  - suitable for mobile deployment.
 - Train the model, tuning hyperparameters such as learning rate, batch size, and epochs for optimal performance.
 - Validate the model on a separate validation set to assess accuracy, robustness, and generalization to different backgrounds and leaf conditions.
 - Save the trained MobileNet V2 model for further integration and testing in the Flutter application.
 - Document training settings, performance metrics, and any observed issues for future reference.

**Comment by lxy1221** [comment]
<img width="3000" height="2400" alt="Image" src="https://github.com/user-attachments/assets/1e8e44cb-6519-454d-90f2-2bcfd1268d7d" />
<img width="3000" height="2400" alt="Image" src="https://github.com/user-attachments/assets/79415875-c628-4f23-a732-643fb1b695ff" />

**Comment by hxw0320** [comment]
New version of confusion matrix:
![Image](https://github.com/user-attachments/assets/a91f8795-d480-4e50-b99b-7af43adf26e9)
